tests/v3/expected/stripe.ts

```diff
@@ -5,106 +5,325 @@
 
 export interface components {
   schemas: {
+    /**
+     * This is an object representing a Stripe account. You can retrieve it to see
+     * properties on the account like its current e-mail address or if the account is
+     * enabled yet to make live charges.
+     *
+     * Some properties, marked below, are available only to platforms that want to
+     * [create and manage Express or Custom accounts](https://stripe.com/docs/connect/accounts).
+     */
     account: {
+      /**
+       * Business information about the account.
+       */
       business_profile?: Partial<
         components["schemas"]["account_business_profile"]
-      >;
+      > | null;
+      /**
+       * The business type.
+       */
       business_type?:
-        | "company"
-        | "government_entity"
-        | "individual"
-        | "non_profit";
+        | ("company" | "government_entity" | "individual" | "non_profit")
+        | null;
```

Comment:
I _believe_ `nullable` has been added, but we can’t be sure before we write unit tests.